# HG changeset patch
# User Wormnest
# Date 1376508596 -7200
#      Wed Aug 14 21:29:56 2013 +0200
# Node ID c076a9121416ca1ff86212db3e8415c21bcdf200
# Parent  559aa17d516181774b9ce140db453962abce0589
Add: VehicleTypesForNews enum to be used for filtering on vehicle type.

diff -r 559aa17d5161 -r c076a9121416 src/news_type.h
--- a/src/news_type.h	Wed Aug 14 21:29:56 2013 +0200
+++ b/src/news_type.h	Wed Aug 14 21:29:56 2013 +0200
@@ -98,6 +98,21 @@
 
 
 /**
+ * The vehicle types that can be filtered.
+ * The order is based on VehicleType except that road vehicles are split into truck and bus
+ * in the same way as in Stations filter.
+ */
+enum VehicleTypesForNews {
+	VTFN_TRAIN    = 1 << 0, ///< Vehicle news item about a train
+	VTFN_TRUCK    = 1 << 1, ///< Vehicle news item about a tuck
+	VTFN_BUS      = 1 << 2, ///< Vehicle news item about a bus
+	VTFN_SHIP     = 1 << 3, ///< Vehicle news item about a ship
+	VTFN_AIRCRAFT = 1 << 4  ///< Vehicle news item about a aircraft
+};
+DECLARE_ENUM_AS_BIT_SET(VehicleTypesForNews)
+typedef SimpleTinyEnumT<VehicleTypesForNews, byte> VehicleTypesForNewsByte;
+
+/**
  * News display options
  */
 enum NewsDisplay {
